cypress/e2e/API/EngageSphere.cy.js

```diff
@@ -0,0 +1,151 @@
+/// <reference types="Cypress" />
+describe('Validações de API', () => {
+    context('Solicitações inválidas de forma elegante', () => {
+      it('Então valido page="invalid" como string', () => {
+        cy.request({
+          method: 'GET',
+          url: 'http://localhost:3001/customers?page=invalid&limit=10&size=All',
```

Comment:
Que tal definir uma variável do tipo `const` chamada `CUSTOMERS_API_URL` no início do arquivo, a qual guarda o seguinte valor: ``${Cypress.env('API_URL')}/customers``?

Dessa forma, essa parte não precisa ser repetida em todos os testes, e em vez disso, tal variável pode ser interpolada com o resto da string.

> **Obs.:** Vale lembrar que pra isso funcionar, você deve definir a `env` `API_URL` à propriedade `e2e` do arquivo de configurações do Cypress (`cypress.config.js`).